### PR TITLE
Add modules documentation

### PR DESCRIPTION
**Pull request by addempsea**

### CODE REVIEW

File: sterling/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+```python
+from sterling.transfer import Transfer
+
+query = {
+        "Referenceid": "01",
+        "RequestType": "01",
+        "Translocation": "01",
+        "ToAccount": "0037514056",
+        "destinationbankcode": "00001"
+    }
```
Comment (akinjide) [inline sterling/README.md:88]: indentation not consistent

Comment (addempsea) [reply sterling/README.md]: done.

File: README.rst [hunk 2]
```diff
@@ -36,9 +36,13 @@ Installation
...
 
-* To get started with the nibss documentation, check out: `NIBSS <https://github.com/enyata/innovation-sandbox-python/blob/master/nibss/README.rst>`_.
+* To get started with the nibss documentation, check out: `NIBSS <https://github.com/enyata/innovation-sandbox-python/blob/master/nibss/README.md>`_.
 
-* To get started with the sterling documentation, check out: `STERLING <https://github.com/enyata/innovation-sandbox-python/blob/master/sterling/README.rst>`_.
+* To get started with the sterling documentation, check out: `STERLING <https://github.com/enyata/innovation-sandbox-python/blob/master/sterling/README.md>`_.
+
+* To get started with the atlabs documentation, check out: `ATLABS <https://github.com/enyata/innovation-sandbox-python/blob/master/atlabs/README.md>`_.
+
+* To get started with the union documentation, check out: `UNION <https://github.com/enyata/innovation-sandbox-python/blob/master/union/README.md>`_.
```
Comment (akinjide) [inline README.rst:45]: Python is README.rst, this will not work.

File: atlabs/README.md [hunk 3]
```diff
@@ -64,7 +65,7 @@ Your Africa's Talking issued virtual phone number.
 
 Phone number to dial
 
-```python3
+```python
```
Comment (akinjide) [inline atlabs/README.md:68]: Did you know python and python3 are different, hence syntax highlighting will be different

**Pull request closed without merging**